forest/src/daemon.rs

```diff
@@ -38,9 +39,12 @@ pub(super) async fn start(config: Config) {
         });
 
     // Initialize keystore
-    let keystore = Arc::new(RwLock::new(
-        PersistentKeyStore::new(config.data_dir.to_string()).unwrap(),
-    ));
+    let mut ks = PersistentKeyStore::new(config.data_dir.to_string()).unwrap();
+    if ks.get("auth-jwt-private").is_err() {
```

Comment:
I think it would be better to make  "auth-jwt-private" a constant somewhere in auth/src/lib.rs instead of reusing it multiple times